enterprise/cmd/frontend/internal/licensing/enforcement/users.go

```diff
@@ -87,7 +87,7 @@ func NewAfterCreateUserHook() func(context.Context, database.DB, *types.User) er
 			return err
 		}
 
-		if info.Plan() == licensing.PlanFree0 {
+		if info.Plan() == licensing.PlanFree0 || info.Plan() == licensing.PlanFree1 {
```

Comment:
How about something like this?

```suggestion
		if info.PlanIsFree() {
```

And hiding some of the logic in this method.